website/frontend/src/components/blocks/SearchForm/SearchForm.module.scss

```diff
@@ -0,0 +1,32 @@
+@use 'assets/scss/variables' as *;
+@use 'assets/scss/breakpoints.scss' as *;
+
+.searchForm {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  gap: 40px;
+  margin-bottom: 20px;
+
+  @include lg {
+    flex-direction: column;
+    gap: 16px;
+  }
+}
+
+.links {
+  display: flex;
+  justify-content: flex-start;
+  align-items: center;
+  flex-wrap: nowrap;
+  gap: 0;
```

Comment:
those are all useless styles imo, since `a` and `button` are inline elements they don't need `links` wrapper